src/main/java/de/uni_tuebingen/ub/nppm/db/EinzelbelegNamenkommentarDB.java

```diff
@@ -0,0 +1,29 @@
+package de.uni_tuebingen.ub.nppm.db;
+
+import java.util.List;
+import org.hibernate.Session;
+import de.uni_tuebingen.ub.nppm.model.*;
+import javax.persistence.criteria.*;
+
+
+public class EinzelbelegNamenkommentarDB extends AbstractBase {
+
+   public static List<EinzelbelegNamenkommentar_MM> getByEinzelbelegId(int einzelbelegId) throws Exception {
```

Comment:
Similar to other comments in EinzelbelegMGHLemmaDB

Reply:
done